content/en/docs/Reference/image_matrix.md

```diff
@@ -16,81 +16,35 @@ These images are pushed to quay.io and are available for installation and upgrad
 
 Kairos release processes generates images based on official container images from popular Linux distributions. If you don't see your preferred distribution, check if [we are already planning](https://github.com/kairos-io/kairos/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Fflavor) support for it or create a new issue.
 
-Below is a list of the available images and their locations on the quay.io registry:
+Below is a list of the container repositories for each flavor:
```

Comment:
This change makes it a lot clearer but I would still at least add a link to the latest release artifacts, so they can also download ISOs or other artifacts